src/node_crypto.cc

```diff
@@ -1749,6 +1751,20 @@ int SSLWrap<Base>::NewSessionCallback(SSL* s, SSL_SESSION* sess) {
 }
 
 
+template <class Base>
+void SSLWrap<Base>::KeylogCallback(const SSL* s, const char* line) {
+  Base* w = static_cast<Base*>(SSL_get_app_data(s));
+  Environment* env = w->ssl_env();
+  HandleScope handle_scope(env->isolate());
+  Context::Scope context_scope(env->context());
+
+  Local<String> line_str = String::NewFromUtf8(env->isolate(),
+      line, NewStringType::kNormal).ToLocalChecked();
```

Comment:
You can use OneByteString() here since the input string is ASCII. That way you don't pay the performance price of decoding UTF-8.

Reply:
As you suggested in the comment, I've changed it to Buffer and then used `.toString('ascii')` on the JS side. WDYT?